be/src/exec/vectorized/lake_meta_scan_node.h

```diff
@@ -0,0 +1,34 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+#pragma once
+
+#include <gen_cpp/Descriptors_types.h>
+
+#include "exec/vectorized/meta_scan_node.h"
+#include "exec/vectorized/lake_meta_scanner.h"
+
+namespace starrocks {
+namespace vectorized {
+
+class LakeMetaScanNode final : public MetaScanNode {
+public:
+    LakeMetaScanNode(ObjectPool* pool, const TPlanNode& tnode, const DescriptorTbl& descs);
+    ~LakeMetaScanNode() override = default;
+
+    Status open(RuntimeState* state) override;
```

Comment:
It seems we don't have to implements prepare/open/get_next method for LakeMetaScanNode